src/battleground/BattlegroundContainer.js

```diff
@@ -0,0 +1,76 @@
+//@flow strict
+
+import * as React from 'react';
+import Battleground from './Battleground.js';
+import HealthBarsField from './HealthBarsField.js';
+import Navbar from '../globalComponents/Navbar.js';
+import Tank from '../tanks/Tank.js';
+import BackendTank from '../tanks/BackendTank.js';
+import { getTank, getEmptyCasusCode } from '../tanks/TankLoader.js';
+import "../Main.css";
+
+type Props = {||};
+
+type State = {
+	playerOneTank: Tank,
+	playerTwoTank: Tank,
+};
+
+class BattlegroundContainer extends React.Component<Props, State> {
+
+	constructor(props: Props) {
+		super(props);
+
+		const blankTank: BackendTank = new BackendTank();
```

Comment:
you can get rid of these 10 lines of code by making playerOneTank and playerTwoTank nullable here and in sub-components